README.md

```diff
@@ -0,0 +1,106 @@
+OpenIddict<br><sub>The OpenID Connect server you'll be addict to.</sub>
+=======
+
+AppVeyor: [![Build status](https://ci.appveyor.com/api/projects/status/d0d8git3o6lqkvbm?svg=true)](https://ci.appveyor.com/project/aspnet-contrib/core)
+
+Travis: [![Build status](https://travis-ci.org/openiddict/core.svg)](https://travis-ci.org/openiddict/core)
+
+### What's OpenIddict ?
+
+OpenIddict aims at providing a simple and easy out of the box solution 
+to implement an OpenID Connect server for ASP.NET 5.
+
+
+### Why an OpenID Connect Server?
+
+With an OpenID Connect Server you can manage all your users using local 
+password or an external identity provider management for all your applications 
+in one central place, with the power to control who can access your API and 
+the information that is exposed to each client. 
+
+
+### How does it work?
+
+OpenIddict, by default, leverages the use of Identity (for user management) and 
+EntityFramework (as an optional store provider).
+
+Adding it to your existing application allows you to register clients and 
+serve them authenticate tokens.
+
+Under the hood it uses [AspNet.Security.OpenIdConnect.Server](https://github.com/aspnet-contrib/AspNet.Security.OpenIdConnect.Server) 
+middleware that works with any standards-compliant OAuth 2.0/OpenID Connect 
+client including the official OpenID Connect client middleware 
+developed by Microsoft.
+
+--------------
+
+## Getting Started
+
+You can find working samples in the [samples](https://github.com/openiddict/core/tree/dev/samples) directory.
+
+Nightly builds can now be found on the [aspnet-contrib](https://github.com/aspnet-contrib) MyGet repository https://www.myget.org/F/aspnet-contrib/api/v3/index.json.
+
+To use OpenIddict Server you need to include OpenIddict as a dependency in your project.json:
```

Comment:
`To use....`  whoa... hold up a sec.

I think this needs to have a list summary of the steps needed to use this nuget package, before any code and text.

eg.

Steps to setup your webapp
1. install-package
2. modify `json.settings`
3. Wire up middleware in `ConfigureServices` (in `startup.cs`)

and now we can go into some code, etc...

Bascially, i need to get an overall feel about how complex it is (via the number of steps), to use this in my app.

Looking at the readme text below this point, i'm not sure if this is _everything_ or just the _hardest_ parts, of the steps.

Remember, i'm blond..... i need to be spoon fed.